cwatm/hydrological_modules/water_demand/water_demand.py

```diff
@@ -584,8 +655,13 @@ def initial(self):
             self.var.ind_efficiency = 1.
             self.var.dom_efficiency = 1.
             self.var.liv_efficiency = 1
-
-
+            
+            # for wastewater package
+            self.var.leakage_wwtC_daily = np.compress(self.var.compress_LR, globals.inZero.copy())
```

Comment:
self.var.compress_LR might not exist

Reply:
Fixed by including an 'if checkOption('includeWaterBodies'):' statement